Assignment2.yaml

```diff
@@ -0,0 +1,408 @@
+swagger: "2.0"
+info:
+  description: "Starwars API."
+  version: "1.0.0"
+  title: "SWAPI"
+host: "swapi.co"
+basePath: "/api/"
+tags:
+- name: "Vehicle"
+  description: "Everything about your Vehicle"
+- name: "People"
+  description: "Everything about your People"
+- name: "Starship"
+  description: "Everything about your Starship"
+- name: "Species"
+  description: "Everything about your Species"
+schemes:
+- "http"
+paths:
+  /Vehicle:
+    get:
+      tags:
+      - "Vehicle"
+      summary: "Find list of vehicles"
+      description: "Returns a List of vehicle"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            type: "array"
+            items:
+             $ref: "#/definitions/Vehicle"
+  /Vehicle/{VehicleId}:
+    get:
+      tags:
+      - "Vehicle"
+      summary: "Find Vehicle by ID"
+      description: "Returns a single vehicle"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "VehicleId"
+        in: "path"
+        description: "ID of Vehicle to return"
+        required: true
+        type: "number"
+      - name: "q"
+        in: "query"
```

Comment:
How are you passing a search query parameter in an API that is already returning you one vehicle? Isn't the query parameter meant for searching in list API? Correct it here.